jwst/resample/resample_spec_step.py

```diff
@@ -83,6 +83,17 @@ def process(self, input):
 
         # Call resampling
         self.drizpars = kwargs
+
+        #  If input is a 3D rateints (which is unsupported) skip the step
+        try:
+            if len((input_models[0][0]).shape) == 3:
+                self.log.warning('Resample spec step will be skipped')
```

Comment:
Should say why the step is being skipped, e.g. "3D input data not supported; step will be skipped"

Reply:
Fixed